.github/workflows/desloppify.yml

```
@@ -41,6 +41,11 @@ jobs:
       - name: Run quality gate
         run: |
           SCAN_OUTPUT=$(desloppify scan --path . --profile ci --no-badge 2>&1)
+ISSUE_CHECK=$(echo "${SCAN_OUTPUT}" | grep -q "0 issues" && echo clean || echo issues)
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)